src/test/java/com/wikia/webdriver/common/remote/notifications/NotificationsOperations.java

```diff
@@ -0,0 +1,21 @@
+package com.wikia.webdriver.common.remote.notifications;
+
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.remote.Utils;
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor(staticName = "using")
+public class NotificationsOperations {
+
+  private final static String NOTIFICATIONS_SERVICE = "notifications/";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)